packages/devtools_app/lib/src/screens/memory/memory_screen.dart

```diff
@@ -43,6 +43,9 @@ class MemoryScreen extends Screen {
 
   @override
   Widget build(BuildContext context) => const MemoryBody();
+
+  @override
+  bool showConsole(bool embed) => true;
```

Comment:
add a TODO noting that we would ideally not show the console when embedded and instead support the same console functionality through the native console in VSCode.

Reply:
Added for both debugger and  memory.

Reply:
Not needed for debugger as IDEs never show the debugger screen as they have their own better more integrated debuggers.